tests/test_nc_to_mmd.py

```diff
@@ -1023,10 +1023,10 @@ def test_alternate_identifier_missing(self):
         )
         md = Nc_to_mmd(os.path.abspath('tests/data/reference_nc.nc'), check_only=True)
         ncin = Dataset(md.netcdf_file)
-        value = md.get_acdd_metadata(
-            mmd_yaml['alternate_identifier'], ncin, 'alternate_identifier'
+        value = md.get_alternate_identifier(
+            mmd_yaml['alternate_identifier'], ncin
         )
-        self.assertEqual(value['alternate_identifier'], None)
+        self.assertEqual(len(value), 0)
```

Comment:
Description of this test should be updated to indicate that it does not verify MMD altID does not exsist, but rather that the test verify that the altID is set to an empty list. As for as I can see in the code, an object is added but it is empty.

Reply:
Great!